sensor/kubernetes/listener/resource_event_handler.go

```diff
@@ -290,6 +290,7 @@ func (k *listenerImpl) handleAllEvents() {
 				},
 			},
 		},
+		Context: k.context,
```

Comment:
This is a bugfix. `SYNC` events did not have an offline context. Which means that if we disconnected shortly before finishing the reconciliation, we could have two `SYNC` events in the pipeline.

Reply:
Good catch!